app/models/post.rb

```diff
@@ -0,0 +1,6 @@
+class Post < ActiveRecord::Base
+  belongs_to :user
+  has_many :comment

```

Comment:
- [ ] [MANDATORY] This association should should be plural, like this `has_many :comments`. This would make sense as a post can have more than one "comments", right?.